docs/esterni/manuali/manuale_acquirente.tex

```diff
@@ -0,0 +1,151 @@
+\\É possibile scambiare le proprie cryptovalute sul network Polygon attraverso uno dei seguenti exchange decentralizzati:
+
+\begin{itemize}
+\item \href{https://www.quickswap.exchange}{QuickSwap};
+\item \href{https://app.sushi.com/}{SushiSwap};
+\item \href{https://polygon.curve.fi/}{Curve}.
+\end{itemize}
+Una volta verificato di possedere una sufficiente quantità di MATIC si può procedere col pagamento. É possibile verificare prima del pagamento l'ammontare in dollari e l'address del venditore (Fig. 2). Infine per concludere il pagamento:
+\begin{itemize}
+\item selezionare "Confirm" (Fig. 2);
+\item confermare la transazione su Metamask.
+\end{itemize}
+
+Dovrà essere pagata anche un piccola somma per coprire i costi di transazione sul network \textit{Polygon}; questa è visualizzabile direttamente da Metamask prima di confermare la transazione.
+\FloatBarrier
+\begin{figure}[!h]
+\centering
+\includegraphics[width=0.5\linewidth]{img/oggetto_in_vendita.png}
+\caption{Acquisto di un prodotto.}
+\end{figure}
+\FloatBarrier
+Inoltre attraverso la sezione "Outgoing" posta nel menù in alto a destra (Fig. 3) sarà possibile visualizzare l'insieme di pagamenti effettuati con Shop Chain attraverso il wallet collegato.
+\FloatBarrier
+\begin{figure}[!h]
+\centering
+\includegraphics[width=0.8\linewidth]{img/visualizzazione_ordini.png}
+\caption{Sezione Outgoing per visualizzazione pagamenti.}
+\end{figure}
+\FloatBarrier
+
+\subsection{Visualizzazione acquisti}
+É possibile selezionare un acquisto per visualizzare i seguenti dati (Fig. 4 riquadro 1):
+\begin{itemize}
+\item \textbf{Price}: indica il prezzo dell'oggetto acquistato;
+\item \textbf{From}: indica l'address del wallet dal quale è stato eseguito il pagamento;
+\item \textbf{To}: indica l'address del wallet del venditore;
+\item \textbf{Opened}: indica la data in cui è stato eseguito l'acquisto;
+\item \textbf{Expired}: indica la data in cui scade l'ordine e in cui i fondi vengono automaticamente sbloccati; si tratta di 14 giorni dopo l'acquisto.
+\item \textbf{Stato dell'ordine}:
+  \begin{itemize}
+    \item \textbf{Open}: l'ordine è ancora aperto e in attesa di conferma;
+    \item \textbf{Confirmed}: l'ordine è già stato confermato attraverso la scannerizzazione del codice QR;
+    \item \textbf{Expired}: l'utente ha chiesto il rimborso poiché non ha ricevuto il pacco entro 14 giorni;
+    \item \textbf{Canceled}: l'ordine è stato rifiutato dall'e-commerce. I fondi utilizzati per il pagamento vengono restituiti all'acquirente nella valuta stabile DAI.
+  \end{itemize}
+\end{itemize}
+\subsection{Gestione acquisti}
+Per ciascun acquisto l'acquirente ha 14 giorni per annullare il pagamento e dunque essere rimborsato con la valuta stabile DAI. Nel caso l'utente voglia annullare il pagamento dovrà selezionare l'opzione "Refund transaction" (Fig. 4 riquadro 2) e dunque ricevere il rimborso.
```

Comment:
E' il contrario, dopo 14 giorni se il timer non è entrato in funzione in automatico questo è il piano B col pulsante. (Il contratto attuale scade dopo 5 secondi xD)